client/home/lib/utilities/components/apiaccess/container.coffee

```diff
@@ -21,14 +21,22 @@ module.exports = class ApiAccessContainer extends React.Component
     TeamFlux.actions.fetchCurrentStateOfApiAccess()
 
 
-  constructor: (props) ->
+  addNewApiToken: ->
+
+    TeamFlux.actions.addApiToken()
+
+
+  switchToggle: (state) ->
+
+    TeamFlux.actions.disableApiTokens state
 
-    super props
 
   render: ->
 
     <View
       apiTokens={@state.apiTokens}
-      apiAccessState={@state.apiAccessState}/>
+      apiAccessState={@state.apiAccessState}
+      switchToggle={@bound 'switchToggle'}
+      addNewApiToken={@bound 'addNewApiToken'}  />
```

Comment:
Let's prefix these with `on`, eg: `onSwitchToggle`, `onAddNewToken` to make them more obvious that they are event bindings.